locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsUtil.java

```diff
@@ -354,6 +355,13 @@ public static String getFirstReachableAddress(ComputeServiceContext context, Nod
     }
     
     public static String getFirstReachableAddress(NodeMetadata node, Duration timeout) {
+        return getFirstReachableAddress(node, timeout, new Predicate<HostAndPort>() {
+            @Override public boolean apply(HostAndPort input) {
+                return Networking.isReachable(input);
+            }});
+    }
```

Comment:
This is the same predicate as in the other file. Reuse, avoid anonymous classes.